acceptance-tests/src/test/java/me/atam/atam4jsampleapp/FailingTestAcceptanceTest.java

```diff
@@ -0,0 +1,14 @@
+package me.atam.atam4jsampleapp;
+
+import me.atam.atam4j.dummytests.FailingTest;
+import me.atam.atam4j.health.AcceptanceTestsHealthCheck;
+import org.junit.Test;
+
+public class FailingTestAcceptanceTest extends AcceptanceTest{
+
+    @Test
+    public void givenSampleApplicationStartedWithFailingTest_whenHealthCheckCalledBeforeTestRun_thenFailureMessageReceived(){
+        applicationConfigurationDropwizardTestSupport = Atam4jApplicationStarter.startApplicationWith(FailingTest.class);
+        checkResponseIsErrorAndWithMessage(AcceptanceTestsHealthCheck.FAILURE_MESSAGE + " 1. Was expecting false to be true", getResponseFromHealthCheck());
```

Comment:
I missed this in a previous PR, but don't like that use of public attributes in `AcceptanceTestsHealthCheck` (example: FAILURE_MESSAGE). 

If these are to be reused, they should be moved elsewhere. Yes, I can't think of of a possible drawback of using this, but I would vote for avoiding it for the sake of consistency atleast.

Just feels unnatural to use public attributes from a class in a different class. Thoughts? @phillbarber